### PR TITLE
add_mixpanel_source_documentation

### PR DESCRIPTION
**Pull request by TrineFN**
Creating Mixpanel Source documentation as per [Notion Task](https://www.notion.so/weld/Documentation-Mixpanel-source-1c641a92ba6f4dd3998da8ea7a70e6c3)

### CODE REVIEW

File: mixpanel-source.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- An active Mixpanel account with permission to access data from projects you would like to sync.
+
+
+## Setup Instructions
+<!-- ### Step 1 - Begin Configuration -->
+In the connector setup form enter the following fields:  
+1. **Label**: enter the destination schema name of your choice (label).
+2. **Project ID**: 
+    1. In Mixpanel, click the Settings icon in the top right corner.
+    ![Alt text](images/mixpanel_2_organization.png?raw=true "Title")
```
Comment (miningape) [inline mixpanel-source.md:14]: You can replace `Alt text` with some text that will appear if it cannot display the image, but it's really not that important XD

Comment (TrineFN) [reply mixpanel-source.md]: thanks @miningape ! I copy pasted a bit too fast.. I removed the "Title" also in latest commit

File: mixpanel-source.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Prerequisites
+To connect Mixpanel to Weld, you need:
+- An active Mixpanel account with permission to access data from projects you would like to sync.
+
+
+## Setup Instructions
+<!-- ### Step 1 - Begin Configuration -->
+In the connector setup form enter the following fields:  
+1. **Label**: enter the destination schema name of your choice (label).
```
Comment (diane-weld) [inline mixpanel-source.md:11]: Just FYI, I think this is something we might need to update in the future, if schema name gets decoupled from the label created here. But that change would require updating to all our FE documentation.

File: mixpanel-source.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+2. **Project ID**: 
+    1. In Mixpanel, click the Settings icon in the top right corner.
+    ![image settings](images/mixpanel_1_settings.png?raw=true)
+    2. Choose Organization Settings
+    ![image organization](images/mixpanel_2_organization.png?raw=true)
+    3. In "Organization Settings", select "Project"
```
Comment (diane-weld) [inline mixpanel-source.md:17]: Minor detail, "Projects" rather than "Project".

File: mixpanel-source.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    ![image add service account](images/mixpanel_7_add_service_account.png?raw=true)
+    4. Fill in the details and create service account
+        - *Name:* Give it a name e.g. "weld_service_account"
+        - *Organization role:* Admin
+        - *Projects:* Select relevant projects
+        - *Project role:* Admin
+        - *Expires:* Never
+    ![image create service accoint](images/mixpanel_8_create_service_account.png?raw=true)
+    4. Click "Create" and you will be shown the Service Account username and secret. Note that once you close this, you will never be able to access the secret again, so be sure to save it somewhere secure
+4. Region: US
```
Comment (diane-weld) [inline mixpanel-source.md:39]: Bloom is US based, but I guess more generally it should be something like 'Select your data warehouse region'.

**Review by diane-weld — COMMENTED** [review]
Content looks great, my main thought is whether all the images might make the documentation too long? 

Taking the hubspot documentation for example, I think it works well that the entire thing fits in one page on my laptop screen. It's hard for me to tell how it will look when it is in the product.

Also, maybe we should talk to the design team about whether there is a consistent look and feel we are going for across our documentation. I'm thinking a loom or gif could also be a concise sleek way of doing it (if possible).

Screen Shot 2022-01-26 at 4 55 37 pm

File: mixpanel-source.md (hunk 2 above)
Comment (diane-weld) [reply mixpanel-source.md:11]: On a side note, I have also put in a linear ticket suggesting "label" be changed to "connection name" 😅 as "label" doesn't seem to be the most descriptive. But nothing for now :)

File: mixpanel-source.md (hunk 3 above)
Comment (TrineFN) [reply mixpanel-source.md]: Good catch.  Fixed in latest commit

File: mixpanel-source.md (hunk 4 above)
Comment (TrineFN) [reply mixpanel-source.md]: Fixed in latest commit

File: mixpanel-source.md (hunk 2 above)
Comment (TrineFN) [reply mixpanel-source.md]: good to know! thanks! :)

**Comment by TrineFN** [comment]
> Content looks great, my main thought is whether all the images might make the documentation too long?
> 
> Taking the hubspot documentation for example, I think it works well that the entire thing fits in one page on my laptop screen. It's hard for me to tell how it will look when it is in the product.
> 
> Also, maybe we should talk to the design team about whether there is a consistent look and feel we are going for across our documentation. I'm thinking a loom or gif could also be a concise sleek way of doing it (if possible).
> 
> Screen Shot 2022-01-26 at 4 55 37 pm

I agree. I think we should talk with the design team about how to optimise this in the future. I think we can still push this as it is now, as it is better than nothing :)

**Review by daryl-daryl — APPROVED** [review]
pushing so we have a guide :)

**Pull request merged**